src/backend/tests/test_crawler_expunger.py

```diff
@@ -254,3 +258,69 @@ def test_expunger_for_record_with_mj_over_21(record_with_mj_over_21):
             status=EligibilityStatus.ELIGIBLE, reason="Eligible now", date_will_be_eligible=date(2001, 3, 3)
         )
     }
+
+@pytest.fixture
+def record_with_mj_under_21_and_traffic_violation():
+    return CrawlerFactory.create(
+        record=YoungDoe.SINGLE_CASE_RECORD,
+        cases={"CASEJD1": CaseDetails.CASE_MJ_AND_TRAFFIC_CONVICTION,})
+
+def test_expunger_for_record_with_mj_under_21_not_blocked_by_traffic(record_with_mj_under_21_and_traffic_violation):
+    expunger_result = Expunger.run(record_with_mj_under_21_and_traffic_violation)
+
+    assert isinstance(record_with_mj_under_21_and_traffic_violation.charges[0].charge_type, MarijuanaUnder21)
+    assert isinstance(record_with_mj_under_21_and_traffic_violation.charges[1].charge_type, TrafficViolation)
+
+    assert expunger_result["CASEJD1-1"] == TimeEligibility(status=EligibilityStatus.ELIGIBLE,
+                                                           reason="Eligible now",
+                                                           date_will_be_eligible=date(1999, 3, 3)) #one year after conviction
+    assert expunger_result["CASEJD1-2"] == TimeEligibility(status=EligibilityStatus.INELIGIBLE,
+                                                           reason="Never. Type ineligible charges are always time ineligible.",
+                                                           date_will_be_eligible=date.max())
+
+def test_expunger_for_record_with_mj_under_21_with_non_traffic_charge():
+    record = CrawlerFactory.create(
+        record=YoungDoe.SINGLE_CASE_RECORD,
+        cases={"CASEJD1": CaseDetails.CASE_MJ_AND_CIVIL_CONVICTION})
+
+    expunger_result = Expunger.run(record)
+
+    assert isinstance(record.charges[1].charge_type, CivilOffense)
+
+    assert expunger_result["CASEJD1-1"] == TimeEligibility(status=EligibilityStatus.ELIGIBLE,
+                                                           reason="Eligible now",
+                                                           date_will_be_eligible=date(1998, 3, 3)+relativedelta(years=1))
+
+    assert expunger_result["CASEJD1-2"] == TimeEligibility(status=EligibilityStatus.INELIGIBLE, reason="Never. Type ineligible charges are always time ineligible.",
+                                                           date_will_be_eligible=date.max())
+
+def test_expunger_for_record_with_mj_under_21_with_non_traffic_charge_not_subject_to_137226():
+    record = CrawlerFactory.create(
+        record=YoungDoe.SINGLE_CASE_RECORD,
+        cases={"CASEJD1": CaseDetails.CASE_MJ_AND_JUVENILE_CONVICTION})
+
+    expunger_result = Expunger.run(record)
+
+    assert isinstance(record.charges[0].charge_type, MarijuanaUnder21)
+    assert isinstance(record.charges[1].charge_type, JuvenileCharge)
```

Comment:
I can not trace back what is meant to be modified in the fixture to produce a JuvenileCharge; it seems like it should be controlled by the case type or violation level, but there were no examples and I spent enough time on it to call for help -- see `./case_html/mj_with_juvenile_offense.html`

Reply:
Just do a `CivilOffense` charge by putting "fugitive" in the charge name.

Reply:
This worked 👍